src/store/api/index.js

```diff
@@ -1,5 +1,6 @@
 import { fallbackApolloClient } from '@/vue-apollo'
 import LogRocket from 'logrocket'
+import settings from '../../../public/settings.json'
```

Comment:
Something's up with this import, I don't think it's being parsed as a static asset - investigating.